tests/node/rpc/structs/DoorStatusTests.js

```diff
@@ -0,0 +1,39 @@
+const SDL = require('../../../config.js').node;
+const DoorStatus = SDL.rpc.structs.DoorStatus;
+
+const Test = require('./../../../Test.js');
+const Validator = require('./../../../Validator.js');
+const BaseStructTests = require('./BaseStructTests');
+
+describe('DoorStatusTests', function () {
+    before(function () {
+        this.create = function () {
+            return new DoorStatus()
+                .setLocation(Test.GENERAL_GRID)
+                .setStatus(Test.GENERAL_DOORSTATUSTYPE);
+        };
+
+        this.getExpectedParameters = function (sdlVersion) {
+            return {
+                [DoorStatus.KEY_LOCATION]: Test.JSON_GRID,
```

Comment:
@ymalovanyi on line 18 does `Test.JSON_GRID` need to be changed to `Test.GENERAL_GRID`?

Reply:
No, because then test will fail. We're checking for parameters here, and Grid is a struct. That is why we are passing JSON parameters of the struct, but not the whole struct itself.